packages/react/src/components/TreeView/TreeNode.tsx

```diff
@@ -57,6 +61,9 @@ export type TreeNodeProps = {
   /**
    * **Note:** this is controlled by the parent TreeView component, do not set manually.
    * TreeNode depth to determine spacing
+   *
+   *    @deprecated The `depth` prop for `TreeNode` has
+   *     been deprecated after the introduction of context. It will be removed in the next major release.
```

Comment:
```suggestion
   * @deprecated The `depth` prop for `TreeNode` has
   * been deprecated after the introduction of context. It will be removed in the next major release.
```